woocommerce/wc-admin
# Updating eway logo in payment suggestions defaults

Fixes part of https://github.com/woocommerce/woocommerce-admin/issues/7359

Updating payment suggestions defaults to include new eway logo. Note that we are now storing this within the repo. Also, this is a companion to the [other PR within WCCOM](11090-gh-Automattic/woocommerce.com) that updates the remote specs. 

### Screenshots

![image](https://user-images.githubusercontent.com/444632/130861980-abc393dd-9c60-4200-b246-a25904104c0c.png)


### Detailed test instructions:

-   Checkout branch
-   Disable marketing suggestions
- Navigate to Home -> Task List -> Set up payments
- Ensure new eway logo is displays, as in screenshot.
